tests/golden/copy-command/copy-command.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "copy an entry" {
+  run coffer create /a
+  assert_output "[SUCCESS] Entry created at '/a'."
+
+  run coffer copy /a /b
+  assert_output "[SUCCESS] Copied '/a' to '/b'."
+
+  run cleanOutput coffer view /
+  cat copy-an-entry.gold | assert_output -
+}
+
+@test "copy a directory" {
+  run coffer create /a/b/c
```

Comment:
Try creating 1 more entry in the `/a` directory, and then check that the output of `coffer copy` mentions _all_ copied entries.

Also try creating 1 entry in a different directory, and check that it _doesn't_ appear in the output of `coffer copy`.

Reply:
Done

Reply:
Forgot about this bit:

> Also try creating 1 entry in a different directory, and check that it doesn't appear in the output of coffer copy.

Reply:
Oops, no you didn't, I see it now, it's a bit further down.

Reply:
No need to run two `coffer copy` commands in that test though, you can just create all 3 entries at once, and then run `coffer copy` once.

Reply:
Done